### PR TITLE
Removed node modules

### PR DESCRIPTION
**Pull request by Nicoale**
Renoved unwanted nodemodules

### CODE REVIEW

File: deposits.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+  function getDetails(){
+    let dname = document.getElementById("name");
```
Comment (sigu) [inline deposits.js:2]: A few questions:
1. What is the value of `dname`?
2. Why is it called `dname`?

File: views/deposits.slim [hunk 2]
```diff
@@ -2,9 +2,10 @@ h2 Deposits
 section id="deposit_form"
   label for="name" Name:
   br
```
Comment (sigu) [inline views/deposits.slim]: Rather than have a `br` tag you can wrap each element into a `div`

**Comment by sigu** [comment]
Update the PR title and description to reflect exactly what has been done

File: deposits.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+    let damount=document.getElementById("amount");
+    fetch("/deposit", {method: "POST", {name: dname, amount: damount}}).then(()=>{})
```
Comment (sigu) [inline deposits.js:4]: What do we do inside the then function?

File: deposits.js (hunk 1 above)
Comment (Nicoale) [reply deposits.js:2]: dname refers to the name entered by the person depositing. I have not added any value to it.

File: views/deposits.slim (hunk 2 above)
Comment (Nicoale) [reply views/deposits.slim]: Alright.

File: deposits.js (hunk 3 above)
Comment (Nicoale) [reply deposits.js:4]: take the name , the amount  and send it to the back.

File: deposits.js (hunk 1 above)
Comment (sigu) [reply deposits.js:2]: Can you console log the value after adding it and paste it here?

Why not just call it `name` ? 🤔

File: deposits.js (hunk 3 above)
Comment (sigu) [reply deposits.js:4]: I see and empty function body in the `then`

File: deposits.js (hunk 1 above)
Comment (Nicoale) [reply deposits.js:2]: it is  a bit confusing considering the id on the slim file is also name. i had to distinguish  one or the other

File: deposits.js (hunk 3 above)
Comment (Nicoale) [reply deposits.js:4]: updating that in a few

File: deposits.js (hunk 1 above)
Comment (sigu) [reply deposits.js]: Okay great

**Comment by Nicoale** [comment]
Photos of my deposits page as at now and the csv file

Screenshot from 2020-04-17 16-12-17
Screenshot from 2020-04-17 16-11-33

**Pull request merged**